### ISSUE TITLE
New maintainer or at least contributor

### ISSUE THREAD

**Issue by ibizaman** [aristidb/aws#278]
Hi!

I’m interested in helping here. I have some work experience with AWS and have some knowledge of Haskell through side projects. I don’t have a lot of time either but I do have interest in helping this package.

**Comment by joeyh** [comment]
@aristidb is seeking a new maintainer for aws, per this tweet:
https://twitter.com/aristidb/status/1455273103278346244?cxt=HHwWiMCwwarLlbIoAAAA

**Comment by joeyh** [comment]
My own interest in aws is limited to the S3 part of it so I would not want to take over full maintenance, but I would be willing to be part of a team of maintainers.

At the moment, there is a pressing need for a release fixing #275.

@aristidb if you granted me write access I'd use it responsibly..

**Comment by joeyh** [comment]
https://github.com/jwiegley is listed as a co-maintainer, but has certainly not been active in a long time. Are you still co-maintaining?

**Comment by aristidb** [comment]
@joeyh Sent you an invite to be "Collaborator", which I think should give you the write access you need.

**Comment by joeyh** [comment]
Thank you @aristidb. I still can't upload aws to hackage, but I'll worry about that once I've gotten a release together.

**Comment by aristidb** [comment]
I tried adding you as a Hackage maintainer on the web interface now, maybe that helps with that.

**Comment by joeyh** [comment]
Thank you @aristidb! I have now released 0.22.1 to hackage.

**Comment by jwiegley** [comment]
@joeyh No, I'm not longer working on this project. You can remove me as a maintainer.
